core/lib/generators/spree/dummy/templates/rails/database.yml

```diff
@@ -33,17 +33,20 @@ production:
 development:
   adapter: postgresql
   database: <%= database_prefix %>spree_development
-  username: postgres
+  username: runner
```

Comment:
Which environment do we actually need for the tests? Is it running in test environment or in development environment?

Another way to solve this is to write our own database.yml on Semaphore: https://semaphoreci.com/docs/languages/ruby/custom-database-yml.html

I added you to the project and you should be able to change the Semaphore configuration.